3rdParty/fuerte/src/Basics/Endian.h

```diff
@@ -0,0 +1,275 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2018 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Christoph Uhde
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef ARANGODB_BASICS_ENDIAN_H
```

Comment:
should use a FUERTE namespace here, in order to avoid conflicts with ArangoDB.

Reply:
ARANGO_CXX_DRIVER_ENDIAN_H

Reply:
Additionally we should check if this code is up-to-date with the code in ArangoDB in lib/Basics/Endian.h.